### ISSUE TITLE
Opencart API 

### ISSUE THREAD

**Issue by mragwa** [opencart/opencart#5118]
Dears, 
I'm working on mobile app based on opencart website and i need to integrate it with REST API. 
I discovered the native API & created an user credentials  by username & API key. 
I used this tutorial : https://code.tutsplus.com/tutorials/explore-rest-apis-in-opencart-20-part-one--cms-25122
but it works only with one user who have the API key i created before!!!.
but what i should to if i want to create an API key to every user registered on the website ?.
basically i need users to login using their normal username/password over REST API. 
Please help me I lost a week searching for that.

**Comment by mragwa** [comment]
@danielkerr why you closed it ???

**Comment by mragwa** [comment]
@danielkerr  plz if you can help me with this issue, i'm stuck on it.

**Comment by jfn0** [comment]
I could help you with that

**Comment by mragwa** [comment]
@jfn0  Thank you for your interest, please if you can send me any tutorial or code snippets. 
I just need to make all users connected to the backend throw REST API.
and I can get user session, orders, ..etc.
what i get according to tuts plus tutorial is to access the API throw API Key not just login using username/password. so what i miss understand is "if two or three users login using the same API key how can i get the active user?". you get it!.

**Comment by jfn0** [comment]
Could you chat with me on Gmail will be more easier

**Comment by jfn0** [comment]
my gmail is jfrederic.nault@gmail.com

**Comment by mragwa** [comment]
Ok, I sent you an invitation

**Comment by antoniocali** [comment]
2020 and still here for same issue :/

**Comment by arunkumar350** [comment]
@antoniocali Yes, We can create the API key when the customer gets registered.

**Comment by antoniocali** [comment]
@arunkumar350 but how access to /login api without having the API Key in first place?
